scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1289,13 +1315,21 @@ private void handleTimeout(DTLSFlight flight) {
 			LOGGER.log(Level.FINE, "Re-transmitting flight for [{0}], [{1}] retransmissions left",
 					new Object[]{flight.getPeerAddress(), max - flight.getTries() - 1});
 
-			flight.incrementTries();
-			sendFlight(flight);
+			try {
+				flight.incrementTries();
+				flight.setNewSequenceNumbers();
```

Comment:
I 'm not sure we must increment sequence numbers on retransmission ?
Does we do that before ?